README.md

```diff
@@ -4,9 +4,93 @@
 
 Codemods for migrating to CanJS 3.0
 
-## transforms.json
+## Overview
+
+`can-migrate-codemods` is large-scale codebase refactoring tool that is partially automated but still requires oversight and intervention. A codemod is a transformation script that parses the [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree) of source code in order to do a code-aware find-and-replace refactor across multiple files. This module contains a CLI with various codemods that transform Can.js 2.x to 3.x. It is intended to help you get started with a Can.js migration from 2.x to 3.x. However, it won't be a complete solution for a seamless migration, but it will get you significantly closer than doing it by hand. Custom codemod transformations can also be used with the can-migrate CLI. 
+
+### What has changed in CanJS 3?
+
+The short version, the core modules have not changed that much however there are a handful of useful new features and a more modular project structure.
+
+For the long version, read more about it [in the migration guide](https://canjs.com/doc/migrate-3.html)
+
+## Install
+
+Install can-migrate from npm:
+  ```shell
+  $ npm install -g can-migrate-codemods
+  ```
+This will make the `can-migrate` command available.
+
+## Usage(CLI)
+
+The CLI provides the following options:
+```
+$ can-migrate [<file|glob> ...]
```

Comment:
as a readme this is fine ... I would put these details in a `@signiture` when it gets put in canjs.com